tests/tests.js

```diff
@@ -54,8 +54,6 @@ exports.defineAutoTests = function () {
 
     // flags
     var isWindows = cordova.platformId === "windows8" || cordova.platformId === "windows";
-    var isWindowsPhone81 = isWindows && WinJS.Utilities.isPhone;
-    var isWP8 = cordova.platformId === "windowsphone";
```

Comment:
Can you confirm this?

```
var isWindows = cordova.platformId === "windows8" || cordova.platformId === "windows";
```
I suspect should become:

```
var isWindows = cordova.platformId === "windows";
```

But then I think the Windows platform can be on a phone? so I believe
```
var isWindowsPhone81 = isWindows && WinJS.Utilities.isPhone;
```
should become
```
var isWindowsPhone = isWindows && WinJS.Utilities.isPhone;
```

Reply:
Good catch(es). I'll take a closer look.

Reply:
done